src/devices/Media/VideoDevice/Devices/Interop.videodev2.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// Disable these StyleCop rules for this file, as we are using native names here.
+#pragma warning disable SA1300 // Element should begin with upper-case letter
+#pragma warning disable SA1307 // Field should begin with upper-case letter
+
+internal partial class InteropVideodev2
```

Comment:
consider following class structure:

```csharp
partial class Interop
{
  public class Videodev2 { /* ... */ }
}
```

Reply:
That does not work here, because the class `Interop` is public while building the binding and resides in a different assembly. Partial classes cannot be merged from two assemblies.

Reply:
Perhaps then consider moving this file over to the Common project? I do feel like having a central place for all of the Interop code makes sense.

Reply:
The problem here is that these classes depend on public(!) enums in the Media project. I could move them, too, but then _they_ would be a bit on the odd side.

Reply:
No, that's what I'm saying: Those enums are public. (I did not test whether they are rightfully so, though)